meetings/bordeaux2018/index.md

```diff
@@ -59,9 +87,21 @@ A preliminary schedule is outlined below. It will be updated once speakers are c
 
 ### Friday 29 June
 
-<tt>Morning</tt> Workshop
+<tt>10:00 - 10:25</tt> [Juniper](https://github.com/lanl-ansi/Juniper.jl): An Open-Source Nonlinear Branch-and-Bound Solver in Julia **Ole Kröger (LANL)**
+
+<tt>10:25 - 10:50</tt> [NEOS.jl](https://github.com/odow/NEOS.jl): A Julia interface to the NEOS Server **Oscar Dowson (University of Auckland)**
+
+<tt>10:50 - 11:20</tt> Coffee Break
+
+<tt>11:20 - 11:45</tt> Automatic reformulation using constraint bridges **Benoît Legat (UCLouvain)**
+
+<tt>11:45 - 12:05</tt> [EAGO](https://github.com/PSORLab/EAGO.jl): A Deterministic Nonconvex Optimization Package for Julia **Matthew Wilhelm (University of Connecticut)**
+
+<tt>12:05 - 12:30</tt> [EMP.jl](https://github.com/xhub/EMP.jl), a package for modelling Extended Mathematical Programming **Oliver Huber (UW Madison)**
+
+<tt>12:30 - 15:00</tt> Lunch + developer collaboration
 
-<tt>15:00</tt> JuMP Tutorial / Developer Collaboration
+<tt>15:00 - 17:00</tt> JuMP Tutorial / Developer Collaboration
```

Comment:
We'll probably encourage most attendees to go to the tutorial, so we don't need to list an alternative activity.